contracts/token/SRC20.sol

```diff
@@ -357,11 +366,11 @@ contract SRC20 is ISRC20, ISRC20Owned, ISRC20Managed, SRC20Detailed, Ownable {
     }
 
     function transferFrom(address from, address to, uint256 value) public returns (bool) {
-        if (_restrictions != ITransferRules(0)) {
-            require(_restrictions.authorize(from, to, value), "Transfer not authorized");
+        if (_rules != ITransferRules(0)) {
+            require(_rules.authorize(from, to, value), "Transfer not authorized");
```

Comment:
Again, looks like `ITransferRules.authorize` and `ITransferRules.doTransfer` should be one method: they do not get called separately in any use-case(?). Indication for this is that with our implementation `doTransfer` basically starts with `authorize` code. Is there any need to separate them?